src/main/java/io/personium/common/auth/token/AbstractOAuth2Token.java

```diff
@@ -211,7 +211,7 @@ public final String getSchema() {
      * returns Role List.
      * @return Role list
      */
-    public final List<Role> getRoles() {
+    public final List<Role> getRoleList() {
```

Comment:
I think this change causes confusing. So it was better to separate to other pull request. But this time OK.